lms/db/__init__.py

```diff
@@ -59,13 +59,10 @@ def update_from_dict(self, data, skip_keys=None):
                 setattr(self, key, data[key])
 
     def __repr__(self):
-        return "{class_}({kwargs})".format(  # pylint: disable=consider-using-f-string
-            class_=self.__class__.__name__,
-            kwargs=", ".join(
-                f"{kwarg}={repr(getattr(self, kwarg))}"
-                for kwarg in self.__table__.columns.keys()  # pylint:disable=no-member
-            ),
```

Comment:
Our roll your own here didn't account for cases where the column name did not match the attribute.

This caused problems with `_public_id` which was worse than just not representing it correctly, as we have a method called `public_id` which when stringified, stringifies the object, which cases infinite recursion.